src/init.jl

```diff
@@ -163,3 +165,41 @@ function __init__()
         @reexport using .UnitfulRecipes
     end
 end
+
+##################################################################
+# COV_EXCL_START
+if bool_env("PLOTS_PRECOMPILE", "true") && bool_env("JULIA_PKG_PRECOMPILE_AUTO", "true")
+    @precompile_setup begin
+        backend()  # get from `Preferences` or env, and initialize backend
```

Comment:
Do we initialize the backend if we don't do this precompilation?

Reply:
No, then you'll have to explicitly call e.g. `gr()`.

Reply:
We at least need to load the preference unconditionally for cache invalidation to work properly.